data/interfaces/default/js/tables/history_table.js

```diff
@@ -265,6 +265,8 @@ history_table_options = {
                 createChildTable(this, rowData)
             }
         });
+
+        $("#history_table_info").append(' with a duration of ' + settings.json.filter_duration + ' (filtered from ' + settings.json.total_duration + ' total)');
```

Comment:
If we're going to have this here please find a way to make it responsive. This is a huge line of text.